kubernetes/k8s-gencfg

```diff
@@ -12,7 +12,15 @@ def parse_file(args, input_file, output_file):
     with open(input_file) as file:
         input_data = file.read()
 
-    output_data = input_data.replace("{{IMAGE_PATH}}", args.image_path).replace("{{IMAGE_TAG}}", args.image_tag)
+    data = input_data.replace("{{IMAGE_PATH}}", args.image_path)
+    data = data.replace("{{IMAGE_TAG}}", args.image_tag)
+
+    # this argument is optional
+    # ~ means null in yaml
+    output_data = data.replace("{{IMAGE_SECRET}}", args.image_secret if (args.image_secret != None) else "~")
+
+
+        
```

Comment:
Would be nice to clean up the unnecessary blank lines here

Reply:
Yep I'll fix that.